python/interpret_text/experimental/introspective_rationale/introspective_rationale_explainer.py

```diff
@@ -185,46 +185,47 @@ def _load_defaults_model_config_dict(self) -> Dict:
             config = {}
         return config
 
-    def explain_local(self, text: str, **kwargs) -> _create_local_explanation:
+    def explain_local(self, X, y=None, name=None) -> LocalExplanation:
         """ Create a local explanation for a given text
-        :param text: A segment of text
-        :type text: str
-        :param kwargs:
-                preprocessor: an intialized preprocessor to tokenize the
-                given text with .preprocess() and .decode_single() methods
-                preprocessor: Ex. GlovePreprocessor or BertPreprocessor
-                hard_importances: whether to generate "hard" important/ non-important rationales
```

Comment:
same thing for "hard_importances" parameter, this seems like something useful for the user to be able to configure?

Reply:
hard_importances, if True, failed because ids did not exist as a variable. Hence I removed it for now since it never actually worked

Reply:
yes, see comment below, it looks like there was a bug in the indentation in the code, so I think hard_importances can be added back, good catch:

it looks like this code was indented incorrectly, as it's unrelated to computing the "hard importances", which should only be related to the zs:
```
        # generate human-readable tokens (individual words)
        seq_len = int(m.sum().item())
        ids = x[:seq_len][0]
```

Reply:
given we did not know about the bug. I will hold off on adding it back. 

https://github.com/interpretml/interpret-text-contrib/issues/99